spec/mailers/late_reservation_mailer_spec.rb

```diff
@@ -15,7 +15,7 @@
     end
     #Test the subject and make sure it is displaying correctly
     it 'renders correct subject' do
-        expect(mail.subject).to eql('Please check your gift reservations!')
+        expect(mail.subject).to eql(Rails.configuration.late_reservation_mailer_subject)
```

Comment:
Consider this: Someone accidentally changes the mailer subject to a value we don't want, like the empty string, or removes it entirely by accident.

This spec would still pass.